test/e2e/tests/simple-send.spec.js

```diff
@@ -14,14 +19,28 @@ describe('Simple send', function () {
         ganacheOptions: defaultGanacheOptions,
         title: this.test.fullTitle(),
       },
-      async ({ driver, ganacheServer }) => {
-        await logInWithBalanceValidation(driver, ganacheServer);
```

Comment:
this method is at the moment defined in helpers file and it has a flavor of a process - this has been defined as the process `unlock-wallet` as a very simple process - it should be expanded to match the different login options we have